project/npda/general_functions/session.py

```diff
@@ -98,41 +98,14 @@ def get_new_session_fields(user, pz_code):
             )
             raise PermissionDenied()
 
-        if Submission.objects.filter(
-            paediatric_diabetes_unit__pz_code=pz_code,
-            submission_active=True,
-            audit_year=audit_year,
-        ).exists():
-            # a submission exists for this PDU for this year
-            submission = Submission.objects.filter(
-                paediatric_diabetes_unit__pz_code=pz_code,
-                submission_active=True,
-                audit_year=audit_year,
-            ).get()
-
-            if submission.csv_file and submission.csv_file.name:
-                can_upload_csv = True
-                can_complete_questionnaire = (
-                    False
-                    if not (user.is_rcpch_audit_team_member or user.is_superuser)
```

Comment:
admin permissions now checked in the `can_use_questionnaire` context helper, allowing us to use `can_complete_questionnaire` to show conditional UI in the patient list